util/CMakeLists.txt

```diff
@@ -27,6 +27,9 @@ if (eigen3_FOUND)
     LIBS -ldrakeUtil
     REQUIRES
     VERSION 0.0.1)
+  add_library(drakeContactConstraintsUtil SHARED drakeContactConstraintsUtil.cpp)
```

Comment:
gah.  john - so sorry that this one has been so much work to merge.  
i don't think this should live in util (they are rigid body manipulator specific so should live over there).  i also don't think that it should be part of the public interface -- the header should not have to be known and the library should not be installed.  

i thought we said that you were just going to add an additional RigidBodyManipulatorContact.cpp which implemented the required public methods from the RigidBodyManipulator class and constrained the scope of these utility methods to RigidBodyManipulatorContact.cpp  (like in the RigidBodyManipulatorURDF.cpp)?